README.md

```diff
@@ -1,18 +1,29 @@
 # .NET Core Samples
 
 This folder contains all the sample code that is part of any topic under
-the .NET Core documentation. There are several different projects that
+the .NET documentation. There are several different projects that
 are organized in sub-folders. These sub-folders are organized similar
-to the organization of the docs for .NET Core.
+to the organization of the docs for .NET.
 
 Some of the articles will have more than one sample associated with them. 
 
 The readme.md file for each sample will refer to the article so that
 you can read more about the concepts covered in each sample.
 
+There are two classes of samples in this repository:
+
+- **Samples** represent programs that demonstrate apoplication or library scenarios. These samples are typically larger, and often use more than one technology, feature, or toolkit.
+- **Snippets** represent small focused examples that demonstrate one feature or syntax. These should be no more than a single screen of code.
+
+In both cases, we verify the samples in our CI system.
```

Comment:
This is not true yet. Just samples are verified and we need @dend to turn them on here on the new repo.

Snippets should be verified as well. But because most of them don't contain a csproj file, we need to find ways to test them.

Reply:
I changed it to reflect the current reality (but still encourage buildable projects moving forward.)